dojo/unittests/test_outpost24_parser.py

```diff
@@ -0,0 +1,24 @@
+from django.test import TestCase
+from dojo.tools.outpost24.parser import Outpost24Parser
+from dojo.models import Test
+
+
+class TestOutpost24Parser(TestCase):
```

Comment:
Does it make sense to have at least one test that check the finding details?

Reply:
You'd think so, but I didn't see any other tests doing that. 🤷‍♂

Reply:
I agree that it doesn't seem to be common in these tests, so maybe it isn't valuable. I did find this example, though, that is doing something along those lines: https://github.com/DefectDojo/django-DefectDojo/blob/ac0502498b467247ad237c5668a868f681d188ca/dojo/unittests/test_checkmarx_parser.py#L165